### DISCUSSION TITLE
Судьба проекта после официального релиза клиента под Linux

### DISCUSSION THREAD

**Discussion by TheKing-OfTime** [cucumber-sp/yandex-music-linux#142] [Q&A] [answered]
Яндекс готовится к официальному релизу приложения для Linux. Что ждёт этот проект после того как релиз состоится?
![image](https://github.com/user-attachments/assets/17ebc127-b46c-470d-81aa-1034d358e61a)
![image](https://github.com/user-attachments/assets/612c8212-988a-4190-aab5-9fc9c262beaa)

**Comment by cucumber-sp** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/c3d8e535-9572-4248-8463-32097aea5f77)
Он навсегда останется в наших сердцах...

**Reply by cucumber-sp** [reply]
Вообще говоря может сделаю патчи интересные для удобства. Но точно вынесу в отдельный проект

**Comment by mopo3ilo** [comment]
Ты уверен, что это не с самого начала релиза этого клиента там есть итак и будет болтаться до скончания веков.
Практика показывает, что Ya достаточно пренебрежительно относится к ОС на ядре Linux и, как следствие, к пользователям этих ОС.

**Reply by TheKing-OfTime** [reply]
> Ты уверен, что это не с самого начала релиза этого клиента там есть итак и будет болтаться до скончания веков.

Да. я уверен. Ибо логика в приложении по обработке линукса появилась с 5.30.3

5.29.3:
![image](https://github.com/user-attachments/assets/f387338e-cd02-47fb-938c-f055b96f362c)
5.30.3:
![image](https://github.com/user-attachments/assets/5bbe8efb-71bb-4ddd-8cf1-59224e0e8a31)

**Reply by mopo3ilo** [reply]
Ну будем надеяться, будем подождать

**Comment by TheKing-OfTime** [comment]
Походу выпустили.
<img width="750" height="140" alt="image" src="https://github.com/user-attachments/assets/464168b2-274f-4dfe-9adb-c8112f37492a" />
Нет под рукой сейчас линукса. Может кто нибудь проверить?
https://music-desktop-application.s3.yandex.net/stable/Yandex_Music_amd64_5.61.1.deb

**Reply by relyadev** [reply]
<img width="1920" height="999" alt="image" src="https://github.com/user-attachments/assets/5aacee61-f82d-492a-b937-641b9a465fb3" />
На арче не работает(А должно?)

**Reply by dipsx** [reply]
ubuntu работает

**Reply by mopo3ilo** [reply]
Ну надо PKGBUILD правильный написать, зависимости там и т.п.
И подменить тупо в этом же проекте.
@cucumber-sp, можно на тебя надеяться? 🙃

**Reply by Crazy-kun** [reply]
А я-то думаю почему новых релизов нет...

**Reply by BartolomeyKant** [reply]
Сегодня заметил, что в aur появился yandex-music-native, в котороом версия уже 5.69.
Поставил. Работает.

**Comment by cucumber-sp** [comment]
Всем привет, пропал с радаров, было много работы. Этот проект заархивирую, на память пусть лежит, не хочется затирать) Создам новый, будет дистрибуция через те же репозитории нативной версии и пересборка под Arch в пакет yandex-music. Скоро будет
